.devcontainer/scripts/container-set.sh

```diff
@@ -20,4 +20,4 @@ useradd vscode --password vscode -m
 usermod -aG sudo vscode
 
 sudo chown -R vscode $HOME
-echo "export PATH=\"$PATH:/root/.local/bin/\"" >> /home/vscode/.profile
+echo "export PATH=\"$PATH:/root/.local/bin:$HOME/.local/bin\"" >> /home/vscode/.profile
```

Comment:
do you really need this?

Reply:
Yes. WIthout this you need to specify for VSCODE task additional options

Reply:
but it is already in the path...